plates.py

```diff
@@ -0,0 +1,65 @@
+import string
+
+def main():
+    plate = input("Plate: ")
+    if is_valid(plate):
+        print("Valid")
+    else:
+        print("Invalid")
+
+def is_valid(s):
+    # print("is_first_two_letter:", is_first_two_letter(s))
+    # print("is_length:", is_length(s))
+    # print("is_middle_letter", is_middle_letter(s))
+    # print("is_first_number", is_first_number(s))
+    # print("is_punctuation", is_punctuation(s))
+    if  (is_first_two_letter(s) and
+        is_length(s) and
+        is_middle_letter(s) and 
+        is_first_number(s) and
+        is_punctuation(s)):
+        return True
```

Comment:
See "the same comment for your pull request" in doc

Reply:
check now

Reply:
There is no change here. Please refactor